src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfMapperService.java

```diff
@@ -635,54 +636,62 @@ private static void checkIAmOwedBox(Map<String, Optional<String>> printFields) {
+                for (String outcome : claimOutcomes) {
+                    switch (outcome) {
+                        case "compensation":
+                            printFields.put(
+                                PdfMapperConstants.Q9_CLAIM_SUCCESSFUL_REQUEST_COMPENSATION,
+                                Optional.of(YES_LOWERCASE)
+                            );
+                            break;
+                        case "tribunal":
+                            printFields.put(
+                                PdfMapperConstants.Q9_CLAIM_SUCCESSFUL_REQUEST_DISCRIMINATION_RECOMMENDATION,
+                                Optional.of(YES_LOWERCASE)
+                            );
+                            break;
+                        case "oldJob":
+                            printFields.put(
+                                PdfMapperConstants.Q9_CLAIM_SUCCESSFUL_REQUEST_OLD_JOB_BACK_AND_COMPENSATION,
+                                Optional.of(YES_LOWERCASE)
+                            );
+                            break;
+                        case "anotherJob":
+                            printFields.put(
+                                PdfMapperConstants.Q9_CLAIM_SUCCESSFUL_REQUEST_ANOTHER_JOB,
+                                Optional.of(YES_LOWERCASE)
+                            );
+                            break;
+                        default:
+                            break;
+                    }
                 }
+
+                printFields.putAll(getCompensationFields(claimantRequests));
             }
-            String claimantCompensationText =
-                caseData.getClaimantRequests().getClaimantCompensationText() == null ? "" :
-                    caseData.getClaimantRequests().getClaimantCompensationText();
-            String claimantCompensationAmount =
-                caseData.getClaimantRequests().getClaimantCompensationAmount() == null ? "" :
-                    caseData.getClaimantRequests().getClaimantCompensationAmount();
-            printFields.put(
-                PdfMapperConstants.Q9_WHAT_COMPENSATION_REMEDY_ARE_YOU_SEEKING,
-                Optional.of(claimantCompensationText + " " + claimantCompensationAmount)
-            );
         }
 
         return printFields;
     }
 
+    private Map<String, Optional<String>> getCompensationFields(ClaimantRequestType claimantRequests) {
+        String claimantCompensationText = claimantRequests.getClaimantCompensationText();
+        claimantCompensationText = claimantCompensationText == null ? "" : claimantCompensationText;
+
+        String claimantCompensationAmount = claimantRequests.getClaimantCompensationAmount();
+        claimantCompensationAmount = claimantCompensationAmount == null ? "" : claimantCompensationAmount;
+
+        return Map.of(
+            PdfMapperConstants.Q9_WHAT_COMPENSATION_REMEDY_ARE_YOU_SEEKING,
+            Optional.of(claimantCompensationText + "\n£" + claimantCompensationAmount)
```

Comment:
Could I suggest trying the platform independent line separator? https://www.baeldung.com/java-string-newline#:~:text=Adding%20Newline%20Characters%20in%20a%20String&text=In%20Windows%2C%20a%20new%20line,the%20end%20of%20our%20string